### DISCUSSION TITLE
Failed on raspberrypi4 reboot as box86 not the default i386 binfmt

### DISCUSSION THREAD

**Discussion by YuriSizuku** [ptitSeb/box86#796] [General]
I have installed the `qemu-static` and `binfmt-support`. 
After a reboot It uses qemu-static as default for i386 exe.
I need to `make install && sudo systemctl restart systemd-binfmt` again to make box86 works. 
I wonder how to set box86 as default binfmt ?

**Comment by ptitSeb** [comment]
Converting this to a discussion
